### ISSUE TITLE
Swagger UI fixes

### ISSUE THREAD

**Issue by knolleary** [flowforge/flowforge#2397]
Having got the swagger UI on to staging, some final tasks have been identified:

1. Revert #2396 and identify a suitable path to load the FF logo from
2. Some validation errors being reported - only just seen them as they appear in a small footnote right at the bottom of the swagger page:

```
{"messages":["attribute paths.'/api/v1/projects/{instanceId}/snapshots/'(post).requestBody.content.'application/json'.schema.items is missing","paths.'/api/v1/teams/{teamId}/audit-log'. Declared path parameter teamId needs to be defined as a path parameter in path or operation level","paths.'/api/v1/teams/{teamId}/members/{userId}'. Declared path parameter teamId needs to be defined as a path parameter in path or operation level","paths.'/api/v1/teams/{teamId}/members/{userId}'. Declared path parameter teamId needs to be defined as a path parameter in path or operation level","paths.'/api/v1/projects/{instanceId}'. Declared path parameter instanceId needs to be defined as a path parameter in path or operation level","paths.'/api/v1/templates/{templateId}'. Declared path parameter templateId needs to be defined as a path parameter in path or operation level","paths.'/api/v1/project-types/{instanceTypeId}'. Declared path parameter instanceTypeId needs to be defined as a path parameter in path or operation level"],"schemaValidationMessages":[]}
```

**Comment by knolleary** [comment]
Fixed via #2402 and #2403

**Comment by MarianRaphael** [comment]
Verified on staging
